using_images/s2i_images/perl.adoc

```diff
@@ -92,3 +92,14 @@ this option while developing or debugging, it is not recommended to turn this
 on in your production environment.
 endif::openshift-origin[]
 |===
+
+ifdef::openshift-origin[]
+== Hot Deploy
+To enable hot deploy in this image, you have to set the `*PERL_APACHE2_RELOAD*` option to "true".
```

Comment:
Is changing the wording from option to environment variable enough, or should I provide an example or link to oc new-app?